Error when trying to build dasein-cloud-google locally

Seems out of sync with core:

```
chriskelner-mbp:dasein-cloud-google Administrator$ mvn clean install -DskipTests
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Dasein Cloud Google 2013.11-gce-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.4.1:clean (default-clean) @ dasein-cloud-google ---
[INFO] Deleting /Projects/twc/dasein-cloud-google/target
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0-beta-1:enforce (enforce-maven) @ dasein-cloud-google ---
[INFO] 
[INFO] --- maven-resources-plugin:2.4.3:resources (default-resources) @ dasein-cloud-google ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Projects/twc/dasein-cloud-google/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:2.0.2:compile (default-compile) @ dasein-cloud-google ---
[INFO] Compiling 51 source files to /Projects/twc/dasein-cloud-google/target/classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.925s
[INFO] Finished at: Thu Jul 17 10:42:36 EDT 2014
[INFO] Final Memory: 16M/226M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.0.2:compile (default-compile) on project dasein-cloud-google: Compilation failure
[ERROR] /Projects/twc/dasein-cloud-google/src/main/java/org/dasein/cloud/google/capabilities/GCEImageCapabilities.java:[34,7] error: GCEImageCapabilities is not abstract and does not override abstract method supportsImageCopy() in ImageCapabilities
```

Also -- please update the pom.xml to use the latest snapshot version for dasein-cloud-test 

```
<version>2014.07.1-SNAPSHOT</version>
```

--- a/pom.xml
+++ b/pom.xml
@@ -98,7 +98,7 @@
 		<dependency>
 			<groupId>org.dasein</groupId>
 			<artifactId>dasein-cloud-test</artifactId>
-			<version>2013.07-SNAPSHOT</version>
+			<version>2014.07.1-SNAPSHOT</version>
 			<scope>test</scope>
 			<optional>false</optional>
 		</dependency>

--- a/src/main/java/org/dasein/cloud/google/capabilities/GCEImageCapabilities.java
+++ b/src/main/java/org/dasein/cloud/google/capabilities/GCEImageCapabilities.java
@@ -82,6 +82,11 @@ public class GCEImageCapabilities extends AbstractCapabilities<Google> implement
     return false;
   }
 
+  @Override
+  public boolean supportsImageCopy() throws CloudException, InternalException {
+	return true;
+  }
+
   @Override public boolean supportsImageSharing() throws CloudException, InternalException{
     return true;
   }

